ppb/engine.py

```diff
@@ -7,15 +7,15 @@
 
 class GameEngine(Engine):
 
-    def __init__(self, first_scene: Type, **kwargs):
+    def __init__(self, first_scene: Type, *, delta_time=None, resolution=None, flags=None, depth=None, log_level=None):
```

Comment:
Please add `**kwargs` as well. If we remove one of these keywords in the future, I'd like to prevent breaking existing code, so going to operate that `**kwargs` is always there.

Reply:
Ahh good point